packages/cli/src/cog/cog.stac.job.ts

```diff
@@ -214,7 +214,10 @@ export class CogStacJob implements CogJob {
 
         if (interval.length == 0) {
             const years = extractYearRangeFromName(imageryName);
-            if (years[0] != -1) {
+            if (years[0] == -1) {
+                const now = new Date().toISOString();
+                interval.push([now, now]);
+            } else {
```

Comment:
Non existent dates are not allow in stac files

Reply:
do we maybe want to default to 1970 then?